.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is VS Code debug config file. I don't know what's our policy regarding committing IDE configuration files to the repo. Happy to include or exclude.

Reply:
we have not been including so far
happy to discuss to revisit

Reply:
Let me exclude for now. If we have more people using VS Code perhaps we can start including.